oop-1-class-constructor/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: OOP 1 What is a class
-nav_order: 0
-has_children: true
-nav_exclude: true
+title: "Lesson 4: OOP: What is a class"
```

Comment:
Let's change this to just `4 - ....` to match this format in the ToC

![image](https://user-images.githubusercontent.com/4563958/134768800-25ac9c54-c52b-4006-8fb4-6ccf877f4e80.png)

Reply:
I think `OOP - Objects, Classes & Constructor` is more descriptive.